enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -208,20 +210,21 @@ func (c *ociContainer) Create(ctx context.Context, workDir string) error {
 	if err := c.createBundle(ctx, pid1); err != nil {
 		return status.UnavailableErrorf("create OCI bundle: %s", err)
 	}
-	// Create container
-	if err := c.invokeRuntimeSimple(ctx, "create", "--bundle="+c.bundlePath(), cid); err != nil {
+	// Create container (discard output as pid 1 is sleep, which inherits and never closes
+	// stdout/stderr)
+	if err := c.invokeRuntimeSimple(ctx, nil, "create", "--bundle="+c.bundlePath(), c.cid); err != nil {
```

Comment:
by discarding stdio here, I think that means that we won't get the error message from `crun` itself if the `create` fails? (the previous code was sending the error message to `os.Stderr` which is not a good long-term solution, but might be helpful during development while we figure out a better solution)

Also, we don't have to figure this out now - but in the future I think it'd be nice if we could separate the stdout, stderr, and exit code from the runtime, vs the user's executed command. This was one of the pain points with both podman and docker - we have to do heuristics on the exit code and `stderr` contents in order to determine whether there was a podman error that can be retried (`UnavailableError`), vs. a normal action failure (ActionResult with exit_code != 0)

Reply:
I implemented a different approach that allows us to get the `create` error message if the call itself fails. It is a bit hacky, but avoids reimplementing `exec.Cmd.Wait()` in terms of `Cmd.Process`.

Separating runtime from container failures is something I could look into, but I don't know how to do this yet in non-`terminal` mode.